app-editors/zoinks/zoinks-0.4.1-r1.ebuild

```diff
@@ -21,9 +21,11 @@ RDEPEND="x11-libs/libX11
 DEPEND="${RDEPEND}
 	nls? ( sys-devel/gettext )"
 
+PATCHES=( "${FILESDIR}/${PN}-cpp14.patch" )
+
 src_prepare() {
+	default
 	sed -i -e 's:-g -Werror::g' configure* || die
-	tc-export CXX
```

Comment:
Yes, it works without tc-export. CMake tells:
`-- The CXX compiler identification is Clang 3.7.1`